wordpress-mobile/WordPress-Android
# Fix Classic Block in the Unsupported Block editor

Description from (https://github.com/wordpress-mobile/WordPress-Android/pull/12679), cherry-picked commit into the new [gutenberg/fix-classic-block-in-unsupported-editor-15.4.1-base](https://github.com/wordpress-mobile/WordPress-Android/tree/gutenberg/fix-classic-block-in-unsupported-editor-15.4.1-base) branch with proper hotfix base ([release/15.4.1](https://github.com/wordpress-mobile/WordPress-Android/tree/release/15.4.1)). 

This PR introduces a fix for the block editor for better support of the Classic Block.

Gutenberg-mobile PR: wordpress-mobile/gutenberg-mobile#2550

It's based on gb-mobile v1.33.1.

To test:

Switch to html mode and write some text in-between two blocks, making sure the text is not inside the block delimiters
Switch back to visual mode
There should be an unsupported block displayed between the blocks the text was inserted
Use the "?" button of the unsupported block and then "Edit block in web browser" to launch the Unsupported Block editor
Notice the text being displayed inside a "Classic" block without an error 🎉
Make some changes to the text and hit the "✔️" icon in the upper right to commit the changes
Switch to html mode again and notice that the text includes the changes made inside the Unsupported Block editor 🎉

PR submission checklist:
